### ISSUE TITLE
CLI not reporting javascript/nodejs dependencies

### ISSUE THREAD

**Issue by sumanthr1** [dependency-check/DependencyCheck#2157]
dependency-check version: 5.2.1

Running a CLI dependency check on a directory with `package.json` file containing the following (among other dependencies):

```
 "dependencies": {
    "express": "4.15.3",
}
```

The report does not have a listing for `express`. In the log file, this `package.json` is discovered as seen by the `Found file` output.

This is the case with both `enableExperimental` flag set or not. Is there a setting or flag that I'm missing? I'm trying to get the dependencies from a `package.json` or `package-lock.json` file.

**Comment by jeremylong** [comment]
Sorry for the delay - is there a `package-lock.json` or `npm-shrinkwrap.json` in the same directory that does not contain `express`?

**Comment by sumanthr1** [comment]
thanks for your response @jeremylong 

I don't believe there was `package-lock.json` or `npm-shrinkwrap.json` - I lost the env I ran that reproduction on.

So, I tried again, making sure there is no `package-lock.json` or `npm-shrinkwrap.json`.
The `package.json` has the following:
```
  "dependencies": {
    "body-parser": "~1.17.2",
    "cookie-parser": "~1.4.3",
    "debug": "~2.2.0",
    "express": "^4.15.3",
    "morgan": "~1.7.0",
    "serve-favicon": "~2.3.0"
  }
```

But the html report does not have a listing for any of these packages with or without vulnerabilities.

dependency-check version: 5.2.1
